cli/cli-client/src/main/java/org/infinispan/cli/commands/Add.java

```diff
@@ -0,0 +1,42 @@
+package org.infinispan.cli.commands;
+
+import org.aesh.command.Command;
+import org.aesh.command.CommandDefinition;
+import org.aesh.command.CommandResult;
+import org.aesh.command.impl.completer.BooleanOptionCompleter;
+import org.aesh.command.option.Argument;
+import org.aesh.command.option.Option;
+import org.infinispan.cli.activators.ConnectionActivator;
+import org.infinispan.cli.completers.CounterCompleter;
+import org.infinispan.cli.impl.ContextAwareCommandInvocation;
+import org.kohsuke.MetaInfServices;
+
+/**
+ * @author Tristan Tarrant &lt;tristan@infinispan.org&gt;
+ * @since 10.0
+ **/
+@MetaInfServices(Command.class)
+@CommandDefinition(name = Add.CMD, description = "Adds/subtracts a value to/from a counter", activator = ConnectionActivator.class)
+public class Add extends CliCommand {
+   public static final String CMD = "add";
+   public static final String COUNTER = "counter";
+   public static final String DELTA = "delta";
+
+   @Argument(completer = CounterCompleter.class)
+   String counter;
+
+   @Option(description = "Does not display the value", completer = BooleanOptionCompleter.class)
+   boolean quiet;
+
+   @Option(required = true, description = "The delta to add/subtract from/to the value")
```

Comment:
Can this be optional? And if so just adds 1?

Reply:
You'd use inc/dec for that